docs/docs/providers/metabase.md

```diff
@@ -72,6 +72,22 @@ resources:
         name: Editor
         permissions:
           - write
+  - type: table
+    policy:
+      id: policy_id
+      version: 1
+    roles:
+      - id: viewer
+        name: Viewer
+        permissions:
+          - all
+  - type: group
+    policy:
+      id: policy_id
+      version: 1
+    roles:
+      - id: member
+        name: Member
```

Comment:
```yaml
permission: 
  - member
```
permission `member` should be required, right? because we have just added the validation in the `validatePermission()`

Reply:
yes, done